pool.go

```diff
@@ -24,11 +24,13 @@ import (
 	"sync"
 
 	"github.com/edgedb/edgedb-go/internal/cache"
+	"github.com/edgedb/edgedb-go/internal/header"
 )
 
 var (
-	defaultMinConns = 1
-	defaultMaxConns = max(4, runtime.NumCPU())
+	defaultMinConns  = 1
+	defaultMaxConns  = max(4, runtime.NumCPU())
+	poolCapabilities = []byte{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xfb}
```

Comment:
Can we form this with a more readable code? Bitmask i64 and then encode it to bytes?

Reply:
updated